static/js/settings_account.js

```diff
@@ -51,22 +51,26 @@ export function update_full_name(new_full_name) {
 }
 
 export function update_name_change_display() {
-    if (!settings_data.user_can_change_name()) {
-        $("#full_name").prop("disabled", true);
-        $(".change_name_tooltip").show();
-    } else {
-        $("#full_name").prop("disabled", false);
-        $(".change_name_tooltip").hide();
+    if ($("#user_details_section").length) {
```

Comment:
I think returning early if the element is not present will be more readable. We follow these approaches in other functions used for live update too.